src/test/frontend/.eslintrc

```diff
@@ -4,6 +4,7 @@
   },
   // Define missing global Angular testing environment variables.
   "globals": {
+    "angular": true,
     "inject": true,
     "module": true,
```

Comment:
Remove module and inject from here? To prevent other from using them (they do not work)